US County Data - pandas.ipynb

```diff
@@ -7,7 +7,8 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    county_state_names = list(zip(list(top_counties['county']), list(top_counties['state'])))
</pre><p>New York City is not a county, this would have been apparent with the fips codes where this one is NaN.</p><p> </p><p>Also the zip is not necessary, you could just print from the dataframe.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CMU-MS-DAS-Modern-Programming-Mini/ClassAssignmentNo4/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>